.github/CODEOWNERS

```diff
@@ -1,3 +1,16 @@
+################
+# Misc
+################
+/*                               @azure/azure-sdk-eng
+/.azure-pipelines/               @azure/azure-sdk-eng
```

Comment:
Eventually want to move this under eng/pipelines.

Reply:
Created:
- #5056